libmscore/stem.cpp

```diff
@@ -397,7 +397,7 @@ QPointF Stem::hookPos() const
       {
       QPointF p(pos() + line.p2());
 
-      qreal xoff = _lineWidth * .5 * mag();
+      qreal xoff = 0.5 * lineWidthMag();
```

Comment:
Same.

Reply:
Same. I can change it by the way. No problem.